state/protocol/badger/snapshot_test.go

```diff
@@ -301,14 +300,125 @@ func TestSealingSegment(t *testing.T) {
 			require.NoError(t, err)
 
 			// sealing segment should be [B2, B3, B4]
-			assert.Len(t, segment, 3)
-			assert.Equal(t, block2.ID(), segment[0].ID())
-			assert.Equal(t, block3.ID(), segment[1].ID())
-			assert.Equal(t, block4.ID(), segment[2].ID())
+			assert.Len(t, segment.Blocks, 3)
+			assert.Equal(t, block2.ID(), segment.Blocks[0].ID())
+			assert.Equal(t, block3.ID(), segment.Blocks[1].ID())
+			assert.Equal(t, block4.ID(), segment.Blocks[2].ID())
 		})
```

Comment:
better to check the results list is empty

Reply:
https://github.com/onflow/flow-go/pull/1533/commits/00a77e58055c9b7fbc3a8d569aa91866610f5646